src/tvheadendApi.test.js

```diff
@@ -0,0 +1,49 @@
+const test = require('ava');
+const sinon = require('sinon');
+const axios = require('axios');
+
+const tvheadendApi = require('./tvheadendApi');
+
+let sandbox;
+let axiosStub;
+
+test.beforeEach(() => {
+  sandbox = sinon.createSandbox();
+  axiosStub = sandbox.stub(axios, "get");
+});
+
+test.afterEach.always(() => {
+  sandbox.restore();
+});
+
+test.serial('returns the right options with auth', t => {
+  const mockConfig = {
+    tvheadend_parsed_uri: 'https://test.test',
+    tvheadend_username: 'foo',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
    tvheadend_username: "foo",
```